.github/workflows/update-foundry-prove.yml

```
@@ -0,0 +1,27 @@
+name: 'Update Foundry prove'
+on: push
+jobs:
+  update-output:
+    name: 'Update Foundry expected output'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: 'Set up Docker'
+        uses: ./.github/actions/with-docker
+        with:
+          tag: kevm-ci-foundry-${{ github.sha }}
+      - name: 'Build kevm-pyk'
+        run: docker exec -u user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make poetry'
+      - name: 'Build Foundry'
+        run: docker exec -u user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make build-foundry -j2'
+      - name: 'Foundry Prove Update'
+        run: docker exec -u user kevm-ci-foundry-${GITHUB_SHA} /bin/bash -c 'make test-integration TEST_ARGS+="-n0 -k test_foundry_prove --update-expected-output"'
+      - name: 'Tear down Docker'
+        if: always()
+        run: |
+          docker stop --time=0 kevm-ci-foundry-${GITHUB_SHA}
```

```suggestion
          tag: kevm-update-output-${{ github.sha }}
      - name: 'Build kevm-pyk'
        run: docker exec -u user kevm-update-output-${GITHUB_SHA} /bin/bash -c 'make poetry'
      - name: 'Build Foundry'
        run: docker exec -u user kevm-update-output-${GITHUB_SHA} /bin/bash -c 'make build-foundry -j2'
      - name: 'Foundry Prove Update'
        run: docker exec -u user kevm-update-output-${GITHUB_SHA} /bin/bash -c 'make test-integration TEST_ARGS+="-n8 -k test_foundry_prove --update-expected-output"'
      - name: 'Tear down Docker'
        if: always()
        run: |
          docker stop --time=0 kevm-update-output-${GITHUB_SHA}
```

This is to make sure we have a unique docker container name, and to not run too many in parallel (cap at 8).